src/csv_reporter.cc

```diff
@@ -137,8 +137,12 @@ void CSVReporter::PrintRunData(const Run & run) {
   // Print user counters
   for (const auto &ucn : user_counter_names_) {
     auto it = run.counters.find(ucn);
-    CHECK(it != run.counters.end());
-    Out << "," << it->second;
+    if(it == run.counters.end()) {
+      Out << ",";
+    }
+    else {
```

Comment:
this dropped 'else' isn't google style. can you run this change through clang-format with Google style selected?